src/components/structures/LeftPanel.tsx

```diff
@@ -119,8 +119,11 @@ export default class LeftPanel extends React.Component<IProps, IState> {
         if (settingBgMxc) {
             avatarUrl = MatrixClientPeg.get().mxcUrlToHttp(settingBgMxc, avatarSize, avatarSize);
         }
+
         const avatarUrlProp = `url(${avatarUrl})`;
-        if (document.body.style.getPropertyValue("--avatar-url") !== avatarUrlProp) {
+        if (!avatarUrl) {
+            document.body.style.removeProperty("--avatar-url");
+        } else if (avatarUrl && document.body.style.getPropertyValue("--avatar-url") !== avatarUrlProp) {
```

Comment:
This can also be shortened now:
```suggestion
        } else if (document.body.style.getPropertyValue("--avatar-url") !== avatarUrlProp) {
```